docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,320 @@
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+In this example we will be setting the modules up in PWM mode. We will be setting up the motors to work as if the quadcopter is set up as in the diagram below.
+
+.. _quad_image:
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/QuadcopterModules.png
+    :align: center
+    :scale: 50
+    :alt: Modules on Quadcopter
+
+    Module Organization on Quadcopter
+
+First, we will set the flight controller specific parameters. ``ARMING_BEHAVE`` can be left as the default “Use Motor Arm Behave”. ``DISARM_TRIGGER`` will be set to “Coast Motors”. This will turn the motor controllers off and allow the motors to spin freely. In ``TELEM_IDS_1`` select 0, 1, 2, and 3 to indicate that the flight controller should request telemetry from those module IDs (because this is a bitset parameter, the actual number that is set will be 15). Set ``VERTIQ_NUM_CVS`` to 4. This means that your IFCI packet will be filled with 4 control signals and that the available control value indices will be 0, 1, 2, and 3. More can be read about this in the :ref:`IFCI documentation<controlling_ifci>`. After these parameters are set, the flight controller will need to be rebooted.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/px4_settings.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+Next, the module parameters will be set. First the ``TARGET_MODULE_ID`` parameter will be set to 0. This means that we are interacting with the module ID 0. When ``TARGET_MODULE_ID`` is set, the Module Params should be reloaded. The ``CONTROL_MODE`` parameter will be set to PWM. The ``MAX_VELOCITY`` and ``MAX_VOLTS`` parameters can be ignored because we are not controlling the motor with velocity or voltage mode. If ``CONTROL_MODE`` is set to one of those, the MAX\_ parameter that it corresponds to needs to be set appropriately. More information about control mode and its the related MAX\_ parameter can be found in the :ref:`Throttle Mode documentation<throttle_mode>`. ``VERTIQ_FC_DIR`` should be set to 2D. All of these parameters will be the same for all of the modules. The last two parameters ``THROTTLE_CVI``, and ``VERTIQ_MOTOR_DIR`` will vary with each module. In our example, ``THROTTLE_CVI`` will be set to match the module ID. Since we are currently setting module ID 0’s settings, we will set ``THROTTLE_CVI`` to 0. ``VERTIQ_MOTOR_DIR`` will be set to 2D Counter Clockwise to match the :ref:`diagram of the quadcopter above<quad_image>`. Ensure the ``VERTIQ_MOTOR_DIR`` parameter matches the direction shown in the diagram for each module ID. This is then repeated for each module ID by selecting a new ``TARGET_MODULE_ID``. The resulting parameters for each module are shown below. Blue boxes are the parameters that match between modules, red are the ones that vary, and yellow is the module ID.
```

Comment:
Might amend the second sentence to say "...we are interacting with the the module with module ID 0"

Later, where it says the "MAX\_" parameter, why do you have the underscore there?

When you discuss throttle CVI, is it worth linking to here to give some more context on what a CVI is?: https://iqmotion.readthedocs.io/en/feature-px4-ifci-tutorial/manual/manual_ifci_control.html

Reply:
Addressed the first and third comment. The MAX_ is in reference to the different control modes respective parameters MAX_VELOCITY and MAX_VOLTS.